pkg/backend/journal.go

```diff
@@ -463,20 +600,48 @@ func NewSnapshotJournaler(
 		serviceLoop = journaler.unsafeServiceLoop
 	}
 
-	go serviceLoop(journalEvents, done)
+	go serviceLoop(ctx, journalEvents, done)
+
+	err := journaler.BeginOperation(engine.JournalEntry{
+		Kind:           engine.JournalEntrySecretsManager,
+		SecretsManager: secretsManager,
+		ElideWrite:     true,
+	})
```

Comment:
just noticed this--won't this conflict with any call to `Write`, since we require `JournalEntryWrite` to be the first entry in the journal if it exists?